stellargraph/mapper/mini_batch_node_generators.py

```diff
@@ -276,6 +276,28 @@ def __len__(self):
         num_batches = len(self.clusters_original) // self.q
         return num_batches
 
+    def _diagonal_enhanced_normalization(self, adj_cluster):
+        # Cluster-GCN normalization is:
+        #     A~ + λdiag(A~) where A~ = N(A + I) with normalization factor N = (D + I)^(-1)
+        #
+        # Expands to:
+        #     NA + NI + λN(diag(A) + I) =
+        #     NA + N(I + λ(diag(A) + I)) =
+        #     NA + λN(diag(A) + (1 + 1/λ) I))
+        degrees = np.asarray(adj_cluster.sum(axis=1)).ravel()
+        normalization = 1 / (degrees + 1)
+
+        # N * A: multiply rows manually
+        norm_adj = adj_cluster.multiply(normalization[:, None]).toarray()
+
+        # N * (1 + lambda) * (diag(A) + I): work with the diagonals directly
```

Comment:
I'm guessing it expands to be the same thing..? but the equation further above makes more sense to me `NA + λN(diag(A) + (1 + 1/λ) I))`, and looks more like the code below:
```
diag = NI
diag_addition = Nλ(diag(A) + (1 + 1/λ)I)
```

Reply:
Oh, yeah, this isn't the same, it was an incorrect old version.